jvm/src/test/scala/fm/common/TestClassUtil.scala

```diff
@@ -0,0 +1,153 @@
+    // -rw-r--r--@ 1 eric  staff  2771 Aug  8 13:27 lorem-ipsum.txt
+
+    testPaths.foreach{ ClassUtil.classpathContentLength(_) should equal(2771) }
+    testFiles.foreach{ ClassUtil.classpathContentLength(_) should equal(2771) }
+  }
+
+  // classpathDirExists
+
+  test("classpathDirExists - directories") {
+    testDirPaths.foreach { ClassUtil.classpathDirExists(_) should equal(true) }
+    testDirFiles.foreach { ClassUtil.classpathDirExists(_) should equal(true) }
+  }
+
+  test("classpathDirExists - files") {
+    testPaths.foreach{ ClassUtil.classpathDirExists(_) should equal(false) }
+    testFiles.foreach{ ClassUtil.classpathDirExists(_) should equal(false) }
+  }
+
+  // classpathFileExists
+
+  test("classpathFileExists - directories") {
+    // Test Directories
+    testDirPaths.foreach{ ClassUtil.classpathFileExists(_) should equal(false) }
+    testDirFiles.foreach{ ClassUtil.classpathFileExists(_) should equal(false) }
+  }
+
+  test("classpathFileExists - files") {
+    testPaths.foreach { ClassUtil.classpathFileExists(_) should equal(true) }
+    testFiles.foreach { ClassUtil.classpathFileExists(_) should equal(true) }
+  }
+
+  // classpathFileExists
+  /*
+  // Directories timestamps get changed every time they get moved to the new resource directory/jar file/etc,
+  // This was used to do a manual test for directories, but commenting out to get tests to pass
+  test("classpathLastModified - directories") {
+    // This is the project-relative path for the directory, and test assumes being ran in the project home
+    val f: File = new File("jvm/src/test/resources/test/classutil")
+    assert(f.isDirectory, s"$f must be a directory (is the working directory the project home?)")
+
+    testDirPaths.foreach{ ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+    testDirFiles.foreach{ ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+  }*/
+
+  test("classpathLastModified - files") {
+    val f: File = new File("jvm/src/test/resources/test/classutil/lorem-ipsum.txt")
+    assert(f.isFile, s"$f must be a file (is the working directory the project home?)")
+
+    testPaths.foreach { ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+    testFiles.foreach { ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+  }
+
+/*
+  test("def findAnnotatedClasses[T <: Annotation](basePackage: String, annotationClass: Class[T], classLoader: ClassLoader = defaultClassLoader): Set[Class[_]]") { }
+*/
+
+  test("findClassNames") {
+    ClassUtil.findClassNames("test.classutil") should equal(testClasses)
+  }
+
+  test("findClasspathFiles") {
+    // Includes recursive file(s)
+    val expectedFiles: Set[File] = {
+      testClasses.map{ _.replace(".", "/") + ".class" } ++ Set("test/classutil/lorem-ipsum.txt", "test/classutil/subdirectory/subfile.txt")
+    }.map{ new File(_) }
+
+    ClassUtil.findClasspathFiles("test.classutil") should equal (expectedFiles)
```

Comment:
Can you add tests for `ClassUtil.findClasspathFiles("")` and `ClassUtil.findClasspathFiles("/")` ?